ingest/defaults/config.yaml

```
@@ -61,6 +61,9 @@ curate:
     - "%Y-%m-%d"
     - "%Y-%m-%dT%H:%M:%SZ"
 
+  # Field containing location information
+  location_field: "location"
```

_nitpicky nit_

```suggestion
  genbank_location_field: "location"
```

Matching the [pathogen-repo-guide config](https://github.com/nextstrain/zika/commit/cc201f4352dfff81a637b00e3b86405e9e615ad3) to keep configs consistent.